core/src/com/unciv/logic/civilization/managers/GreatPersonManager.kt

```diff
@@ -16,9 +17,8 @@ class GreatPersonManager : IsPartOfGameInfoSerialization {
     lateinit var civInfo: Civilization
 
     /** Base points, without speed modifier */
-    private var pointsForNextGreatPerson = 100
     var pointsForNextGreatGeneral = 200
-
+    private var pointsForNextGreatPerson = Counter<String>()
```

Comment:
This I haven't considered. I'll try to fix tomorrow when I get the chance to. That said, I'm still new to kotlin and in general this codebase, so don't mind the silly question: how does `setSerializer` work? Haven't had to make use of something like this so genuinely asking in case for this situation and in case I have to use it later

Reply:
Not going to lie, that's all going above my head and the gdx documentation feels lacking in my eyes to understand at my level. For now I'll just add back in the variable and add in an if to cover the difference. It probably is not as optimized as a setSerializer, but I'd need help know how to convert the empty string key to one